brayns/common/scene/Scene.cpp

```diff
@@ -46,8 +46,14 @@ Scene::Scene(
 
 Scene::~Scene( )
 {
+    reset();
```

Comment:
Not really. It is necessary in the inherited classes, for the various engines, but definitely not required in the Abstract Scene class. Removed.